ebuild-writing/common-mistakes/text.xml

```diff
@@ -477,8 +491,42 @@ at least when the ebuild hits the stable branch.
 </p>
 </body>
 </subsection>
-</section>
 
+<subsection>
+<title>Not using latest EAPI</title>
+<body>
+<p>
+Often, user-submitted ebuilds do not use the latest EAPI and may even
+be several versions behind. EAPIs bring new helper functions and improved
+methods for completing common tasks. It's recommended that you port submitted
+ebuilds to the latest EAPI possible (subject to eclass constraints) to improve
+the ebuild's quality and ease review.
```

Comment:
Hm. Is "you port" the best approach? Maybe better to ask the user to update his submission, otherwise you'll have the same problem next time again.

Reply:
I've changed it to "It's recommended that submitted ebuilds are..."